compliance/compliance.go

```diff
@@ -534,22 +547,22 @@ func AddIPv4EntryRandom(c *fluent.GRIBIClient, t testing.TB, _ ...TestOpt) {
 		chk.IgnoreOperationID(),
 	)
 
-	// TODO(robjs): add gNMI subscription using generated telemetry library.
+	flushServer(c, t)
```

Comment:
If the compliance test wrapper does the flush, do we still need to do it here?

Reply:
Good discussion point -- I think I had added here to ensure that it didn't need to be repeated at each caller -- however, with the observation that we might want to run some of the tests on a server that doesn't support flush - I agree that moving it into the runner sounds better. I'll make that change now.

Reply:
Ah, actually, I remember why I made this change before -- the problem is that there is not a single mode of `Flush` that we can call for all scenarios. If the server is operating in `ALL_PRIMARY` mode currently then overriding the election ID is not allowed.  This means that we need to mark somewhere what the election mode is -- or make a change to the gRIBI API to say that flush with override should always be allowed.